<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="PassComp">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>30</y>
      <width>171</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Medium</family>
      <pointsize>20</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="PassLengthSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="CopyButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QWidget" name="excludeChar" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>691</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic Medium</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Exclude Symbols:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_30">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>@</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>{ }</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_31">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_32">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>[ ]</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_33">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_34">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>( )</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_35">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_36">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_37">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>\</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_38">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_39">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>|</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_40">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;&quot;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_41">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_42">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_43">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_44">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic Medium</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>~</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_45">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic Medium</family>
       <pointsize>23</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>`</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_46">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_47">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic Medium</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>$</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_48">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>!</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_49">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_50">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_51">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_52">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_53">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>' '</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_54">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_57">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_55">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_58">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_56">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>16</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">&amp;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="PassLengthLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Length: (1-64)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PassCompLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Complexity: (1-3)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="GenPass">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>261</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate Password</string>
    </property>
   </widget>
   <widget class="QSlider" name="PassCompSlider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="PassLength">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>161</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Medium</family>
      <pointsize>20</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="GenPassText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
